packages/sitecore-jss-nextjs-editing-host/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@sitecore-jss/sitecore-jss-nextjs-editing-host",
+  "version": "16.0.0-canary.1",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "npm run clean && tsc",
+    "clean": "del-cli dist types",
+    "lint": "tslint -p . -c ./tslint.json \"src/**/*.ts\"",
+    "test": "mocha --require ts-node/register \"./src/**/*.test.ts\"",
+    "prepublishOnly": "npm run build",
+    "coverage": "nyc --require ts-node/register npm test"
```

Comment:
@ambrauer Currently `npm run coverage` is not working,
you can fixed it by replacing to `nyc -npm test`. Please, check `Uncovered lines` and add tests, if you can cover these lines.
![image](https://user-images.githubusercontent.com/55081915/102229876-e9c6e300-3ef4-11eb-9555-34dd63dc7bfc.png)

Reply:
@sc-illiakovalenko Fixed coverage.